src/nl/rubensten/texifyidea/settings/labeldefiningcommands/EditLabelDefiningCommand.kt

```diff
@@ -0,0 +1,104 @@
+package nl.rubensten.texifyidea.settings.labeldefiningcommands
+
+import com.intellij.openapi.ui.DialogWrapper
+import com.intellij.openapi.ui.ValidationInfo
+import com.intellij.ui.components.JBLabel
+import java.awt.Color
+import java.awt.GridBagConstraints
+import java.awt.GridBagLayout
+import java.awt.event.WindowAdapter
+import java.awt.event.WindowEvent
+import javax.swing.*
+import javax.swing.text.PlainDocument
+
+/**
+ * window to enter the information about a label defining command
+ */
+class EditLabelDefiningCommand(cmdName: String, position: Int, labelAnyPrevCommandStatus: Boolean) :
+        DialogWrapper(null, false) {
```

Comment:
Wrap Kotlin style. Also `cmdName` -> `commandName`

```kotlin
class EditLabelDefiningCommand(
        commandName: String,
        position: Int,
        labelAnyPreviousCommandStatus: Boolean
) : DialogWrapper(null, false) {
```

Reply:
fixed in commit ed34f48cd6dd8da4a8ed3d61d6cf44599f4bc7dd